### PR TITLE
Upgrade node version of Github Action

### PR DESCRIPTION
**Pull request by llFreetimell**
This commit upgrades `node` version of Github Action from 12 to 16,
and apply changes for `package-lock.json`.

ONE-vscode-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

---

Parent Issue : #286

### CODE REVIEW

**Review by YongseopKim — APPROVED** [review]
LGTM

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,3090 @@
 {
   "name": "one-vscode",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "name": "one-vscode",
+      "version": "0.0.1",
```
Comment (mhs4670go) [inline package-lock.json]: Seems duplicated. But they were automatically generated, weren't they?

**Review by mhs4670go — APPROVED** [review]
LGTM

File: package-lock.json (hunk 1 above)
Comment (llFreetimell) [reply package-lock.json]: Yeap, they were automatically generated so I don't know why they are duplicated :(

**Pull request merged**